components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,138 @@
+7. Once the test is successful, select **Create Data Feed** to save your Data Feed configuration
+
+### Edit a Data Feed Instance
+
+To edit a Data Feed instance in Chef Automate:
+
+1. In _Data Feeds_, select the Data Feed name to open its detail page
+1. Edit the Data Feed name or URL
+1. Use the **Save** button to save your changes
+
+### Delete a Data Feed Instance
+
+To delete a Data Feed instance in Chef Automate:
+
+1. In _Data Feeds_, select **Delete Data Feed** from the menu at the end of the table row
+1. Select **Delete Data Feed** to confirm permanent deletion of this Data Feed
+
+## Configuring Global Data Feed Behavior
+
+{{< info >}}
+The settings in `config.toml` apply across all configured Data Feed instances.
+{{< /info >}}
+
+Modify Data Feed behavior with configuration settings in `config.toml`. Using the Chef Automate command-line tool, navigate to `/hab/svc/data-feed-service/config/config.toml`.
+Change one or more configuration settings to reflect the desired global Data Feed behavior:
+
+- Update the `feed_interval` setting to change the interval for the Data Feed collection. The default value is four hours
+- Update the `node_batch_size` setting to change the number of sets of node data sent in each individual batch to your endpoint. The default value is 50 nodes
+- Use the `updated_nodes_only` setting to determine what data to include in each export. The default setting is `true`, which causes the aggregation of only the *changed* data of updated nodes since the last export. Set `updated_nodes_only` to `false` and it aggregates *all* data of updated nodes since the last export
+- To reduce the IP address range for the collected and processed node data, update the `disable_cidr_filter` setting to `false` **and** update the `cidr_filter` setting to cover the required IP address range. For example, you may wish to send only production or test node traffic
+- The `accepted_status_codes` setting is an array of HTTP status codes that the Data Feed Service will treat as success if returned by the 3rd party endpoint. If the status code is not in the `accepted_status_codes` list then an error will be logged
+
+### Config.toml Example
+
+```toml
+    [service]
+
+    host = "localhost"\
+    port = 14001\
+    feed_interval = "4h"\
+    asset_page_size = 100\
+    reports_page_size = 1000\
+    node_batch_size = 50\
+    updated_nodes_only = true\
+    disable_cidr_filter = true\
+    cidr_filter = "0.0.0.0/0"\
+    external_fqdn = ""\
+    accepted_status_codes = [ 200, 201, 202, 203, 204 ]
+```
+
+To debug any issues with the Data Feed Service in Chef Automate, update the following section in `config.toml` by changing the `log_level` value to "debug":
+
+```toml
+    [log]
+    log_format = "text"
+    log_level = "info"
+```
+
+## Data Feed Output Syntax and Details
+
+The outputted data from Data Feed consists of line-separated JSON strings.
+Each line represents the data for one node.
+Each line is of the format:
```

Comment:
Your use of "line" throughout here (i.e. "line-separated" and "each line") is confusing. Your example clearly spans almost 20 "lines", so each line does not represent the data for one node. Correct me if I am wrong, but the only way I could reconcile this would be to add:
```
(The example below has been exploded into one-property-per-line just for readability,
but actual output will have the entire JSON object on a single line.)
```

Reply:
Fixed.